src/corg/developer-environment-setup/getting-the-nuxeo-source-code.md

```diff
@@ -225,74 +225,42 @@ Starting from version 5.9.2, Nuxeo requires Maven <span style="color: rgb(34,34,
 
 ### Getting the Development Branch
 
-{{#> panel type='code' heading='Clone in Read-Only mode'}}
-
-```
-git clone git://github.com/nuxeo/nuxeo.git
-cd nuxeo
-python clone.py master -a
-
-```
-
-{{/panel}}{{#> panel type='code' heading='Clone in Read+Write mode'}}
+{{#> panel type='code' heading='Clone Nuxeo Platform repositories'}}
 
 ```
 git clone git@github.com:nuxeo/nuxeo.git
 cd nuxeo
 python clone.py master -a
-
 ```
 
-{{/panel}} {{#> callout type='note' }}
+{{/panel}}
+
+{{#> callout type='note' }}
 
-*   clone.py can be run with Python 2.7.x but not 3.x versions.
+*   Python 2.7 is required. The scripts are not compliant with Python 3.x versions.
 *   Users of MSysGit should run clone.py with option --no_drive_mapping or -n as MSysGit is not able to access dynamically mounted drives. This option is available as of version 5.7.1.
 
-{{/callout}} {{#> callout type='note' }}
+{{/callout}}
+
+{{#> callout type='note' }}
 
 Contributing to Nuxeo **does not** require cloning Nuxeo source code: see [Contributing to Nuxeo]({{page space='nxdoc' page='contributing-to-nuxeo'}})
 
 {{/callout}}
 
-### Import Nuxeo Source Code in Eclipse
-
-Of course for the dependencies to be resolved in Eclipse they will have to be loaded first in your Maven repository, which you can do by performing at least one build first:
+After having cloned the Nuxeo repository, you need to run a full build in Maven to download all the dependencies.
 
 {{#> panel type='code' heading='Full build Nuxeo without running tests'}}
 
 ```
-export MAVEN_OPTS="-Xmx4096m -Xms1024m -XX:MaxPermSize=512m"
-mvn -DskipTests install -Paddons
-(cd nuxeo-distribution ; mvn -DskipTests install)
+export MAVEN_OPTS="-Xmx4096m -Xms1024m"
+mvn -DskipTests install -Paddons,distrib
 ```
 
 {{/panel}}
 
-If you want to open/edit the projects in Eclipse do the following at the very root of the Git repository:
-
-{{#> panel type='code' heading='Generate Eclipse\'s project files'}}
-
-```
-# For details, see Maven Eclipse Plugin documentation: https://maven.apache.org/plugins/maven-eclipse-plugin/eclipse-mojo.html
-mvn eclipse:clean eclipse:eclipse -Paddons,distrib [-DdownloadSources=true] [-DdownloadJavadocs=true] [-Declipse.useProjectReferences=true] [-Declipse.workspace=/path/to/eclipse/workspace] -fae -nsu -T 1C
-# Linux and OS X users should then run the following script to make Eclipse use different build directories than Maven:
-./fixeclipse
-# A cross-platform script is also available for Windows users:
-# python scripts/fixeclipse.py
```

Comment:
This information wasn't removed, it has been kept and moved into the Eclipse Setup page in commit a70280b, see the [line 72 - 121](https://github.com/nuxeo/doc.nuxeo.com-content/blame/a70280bb91699f2e6db0cd218f80afddbb732975/src/corg/developer-environment-setup/eclipse-setup.md#L72-L121). Actually, I reworked the section, description previously embedded in the code block are now extracted as a paragraph. I guess that's why the git-diff cannot show the history intuitively.